src/highdicom/sc/sop.py

```diff
@@ -367,3 +367,158 @@ def __init__(
             self.PixelData = encapsulate([encoded_frame])
         else:
             self.PixelData = encoded_frame
+
+    @classmethod
+    def from_study_dataset(
```

Comment:
I find the name `study_dataset` confusing, since a data set encodes an instance. Can we call it `reference_dataset` or `ref_dataset` instead?

Reply:
Totally agree. I knew `study_dataset` wasn't great but was drawing a blank on something better and was hoping you could suggest one. Changed the method and its parameter to `ref_dataset` as suggested in 5cce71d.